
  My custom patch ready-to-apply to have an "usable" houston.
  
  Contains:
  - Simplelogger
  - Listen localhost only
  - Queue-Retry every 5 seconds.

  Patch based on "houston-02.01.12.00" aka
  "3c61ef7ded53d6340001d2b56cd829d2ae450580" from 2023-01-12.

diff --git a/houston-process/pom.xml b/houston-process/pom.xml
index fff9c178..960c0098 100644
--- a/houston-process/pom.xml
+++ b/houston-process/pom.xml
@@ -20,6 +20,26 @@
     </properties>
 
     <dependencies>
+      <dependency>                              <!-- TODO: Remove -->
+        <groupId>org.slf4j</groupId>            <!-- TODO: Remove -->
+        <artifactId>slf4j-api</artifactId>      <!-- TODO: Remove -->
+        <version>1.7.25</version>               <!-- TODO: Remove -->
+      </dependency>                             <!-- TODO: Remove -->
+      <dependency>                              <!-- TODO: Remove -->
+        <groupId>org.slf4j</groupId>            <!-- TODO: Remove -->
+        <artifactId>slf4j-simple</artifactId>   <!-- TODO: Remove -->
+        <version>1.7.25</version>               <!-- TODO: Remove -->
+      </dependency>                             <!-- TODO: Remove -->
+      <dependency>                              <!-- TODO: Remove -->
+        <groupId>org.slf4j</groupId>            <!-- TODO: Remove -->
+        <artifactId>jcl-over-slf4j</artifactId> <!-- TODO: Remove -->
+        <version>1.7.25</version>               <!-- TODO: Remove -->
+      </dependency>                             <!-- TODO: Remove -->
+      <dependency>                              <!-- TODO: Remove -->
+        <groupId>org.slf4j</groupId>            <!-- TODO: Remove -->
+        <artifactId>jul-to-slf4j</artifactId>   <!-- TODO: Remove -->
+        <version>1.7.25</version>               <!-- TODO: Remove -->
+      </dependency>                             <!-- TODO: Remove -->
         <!-- project -->
         <dependency>
             <groupId>ch.post.it.paisa.houston</groupId>
diff --git a/houston-process/src/main/java/ch/post/it/paisa/houston/process/main/Deployer.java b/houston-process/src/main/java/ch/post/it/paisa/houston/process/main/Deployer.java
index ee7d8b02..b28ae8d6 100644
--- a/houston-process/src/main/java/ch/post/it/paisa/houston/process/main/Deployer.java
+++ b/houston-process/src/main/java/ch/post/it/paisa/houston/process/main/Deployer.java
@@ -309,7 +309,7 @@ public class Deployer {
                 qc.add(new QueueConfiguration().withPattern("timetable-situation-trips-for-aws").withRetryIntervals(standardDelays)
                     .withEnqueueDelayMillisPerSize(10).withEnqueueMaxDelayMillis(100));
                 // All other queues (typically to backend services) with a slow-down pattern after failed delivery
-                qc.add(new QueueConfiguration().withPattern(".*").withRetryIntervals(standardDelays));
+                qc.add(new QueueConfiguration().withPattern(".*").withRetryIntervals(5/*TODO revert*/));
 
                 RedisquesConfiguration redisquesConfig = RedisquesConfiguration.with()
                         .address(Address.redisquesAddress())

